test/e2e/runner/main.go

```diff
@@ -290,6 +300,9 @@ func NewCLI() *CLI {
 		"Comma-separated list of node names to send load to. Manifest option send_no_load will be ignored.")
 	loadCmd.PersistentFlags().BoolP("internal-ip", "i", false,
 		"Use nodes' internal IP addresses when sending transaction load. For running from inside a DO private network.")
+	loadCmd.PersistentFlags().IntP("duplicate-num-nodes", "", 0,
```

Comment:
The name of the flag is not very clear to me but I don't have an alternative name to propose.

Reply:
How about `num_copies`?  No need to change it if you don't like it.

Reply:
Renamed to `num-nodes-per-tx` . Although that is also not accurate. Implies that **each** tx gets duplicated, while that is not true.